Microsoft.Azure.Cosmos/src/Batch/ItemBatchOperation.cs

```diff
@@ -189,57 +188,10 @@ internal static Result WriteOperation(ref RowWriter writer, TypeArgument typeArg
                     }
                 }
 
-                if (options.Properties != null)
+                r = operation.RequestOptions.WriteRequestProperties(ref writer, pkWritten);
```

Comment:
Couple lines above there is code that simplifies the reference to `options`
```suggestion
                r = options.WriteRequestProperties(ref writer, pkWritten);
```

Reply:
ack, fixing this in the next iteration.